internal/pkg/agent/application/coordinator/coordinator.go

```diff
@@ -1224,14 +1254,15 @@ func (c *Coordinator) filterByCapabilities(comps []component.Component) []compon
 // It returns any resulting errors as a multierror, or nil if no errors
 // were reported.
 // Called on the main Coordinator goroutine.
-func collectManagerErrors(timeout time.Duration, varsErrCh, runtimeErrCh, configErrCh chan error) error {
+func collectManagerErrors(timeout time.Duration, varsErrCh, runtimeErrCh, configErrCh, upgradeMarkerWatchErrCh chan error) error {
 	var runtimeErr error
 	var configErr error
 	var varsErr error
+	var upgradeMarkerErr error
 	// in case other components are locked up, let us time out
 	timeoutWait := time.NewTimer(timeout)
 	defer timeoutWait.Stop()
-	var returnedRuntime, returnedConfig, returnedVars bool
+	var returnedRuntime, returnedConfig, returnedVars, returnedUpgradeMarker bool
```

Comment:
[NIT]
This could be more consistent

```go
	var runtimeErr, configErr, varsErr, upgradeMarkerErr error
	var returnedRuntime, returnedConfig, returnedVars, returnedUpgradeMarker bool

	// in case other components are locked up, let us time out
	timeoutWait := time.NewTimer(timeout)
	defer timeoutWait.Stop()
```

Reply:
Fixed in 1cd2e07938.